### PR TITLE
Remove current vcpu

### PR DESCRIPTION
**Pull request by JasonChenCJ**

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

**Comment by dongyaozu** [comment]
@JasonChenCJ it is time to rebase now...  VPID patch is in.

File: hypervisor/arch/x86/trusty.c [hunk 1]
```diff
@@ -133,9 +135,13 @@ static void create_secure_world_ept(struct vm *vm, uint64_t gpa_orig,
 	vm->sworld_control.sworld_memory.base_hpa = hpa;
 	vm->sworld_control.sworld_memory.length = size;
 
-	invept(vm->current_vcpu);
-	invept(vm0->current_vcpu);
+	foreach_vcpu(i, vm, vcpu) {
```
Comment (dongyaozu) [inline hypervisor/arch/x86/trusty.c]: why we need to flush EPT? with current vpid solution, we don't need to .. Not?

Comment (JasonChenCJ) [reply hypervisor/arch/x86/trusty.c]: @dongyaozu , there is EPT mapping change here for both UOS normal world and SOS.

Comment (dongyaozu) [reply hypervisor/arch/x86/trusty.c]: ok, SOS removed some mappings, so we need flush.
for UOS, we uses different EPTP, so they won't conflict with each other, why we need to flush?

Comment (JasonChenCJ) [reply hypervisor/arch/x86/trusty.c]: For UOS, we are also removing trusty memory from normal world. We are just using different EPTP for normal world and trusty world.

**Comment by lijinxia** [comment]
Verified+1

**Pull request merged**